code/modules/mob/living/carbon/human/species/_species.dm

```diff
@@ -1028,6 +1028,7 @@ It'll return null if the organ doesn't correspond, so include null checks when u
 	return TRUE
 
 /datum/species/proc/spec_hitby(atom/movable/AM, mob/living/carbon/human/H)
+	SEND_SIGNAL(H, COMSIG_ATOM_HITBY, AM)
```

Comment:
Can you put this at the lowest level of `/atom/proc/hitby(atom/movable/AM, skipcatch, hitpush, blocked, datum/thrownthing/throwingdatum)` instead of at species? That would be more accurate to its name

Reply:
Tested this, human mobs dont actually call parent atom proc and refactoring this would be out of scope of this PR tbh

Reply:
Please rename this signal then, the name should be accurate.

Reply:
The name will need to be changed for this PR